deps/wazuh_testing/wazuh_testing/migration_tool/utils.py

```diff
@@ -0,0 +1,187 @@
+        configuration (list): Configuration sanitized.
+    '''
+    for configurations_obj in configuration:
+        configurations_list = configurations_obj['configurations']
+        for config_obj in configurations_list:
+            for key in config_obj:
+                config_obj[key.lower()] = config_obj.pop(key)
+
+    return configuration
+
+
+def validate_json_against_schema(json_document, schema):
+    '''Validate a JSON document under the given schema.
+    Args:
+        json_document (str): Path of the JSON document to be validated
+        schema (str): Path of the CVE5 Schema by default.
+    Returns:
+        result (bool): False if the validation thrown an error, True if not.
+        error (str): Error in the JSON document.
+    '''
+    schema = read_json_file(schema)
+
+    try:
+        validate(instance=json_document, schema=schema)
+    except ValidationError as err:
+        return False, err.message
+
+    return True, ''
+
+
+def validate_against_delta_schema(_elements):
+    '''Wrapper function
+    '''
+    _result = True
+    _errors = []
+    for cve in _elements:
+        _result, _error = validate_json_against_schema(cve, DELTA_SCHEMA_PATH)
+        if _result is False:
+            _errors.append(_error)
+
+    return _errors
+
+
+def validate_against_cve5_schema(_elements):
+    '''Wrapper function
+    '''
+    _result = True
+    _errors = []
+
+    for cves in _elements:
+        data = json.loads(cves['data_blob'])
+        _result, _error = validate_json_against_schema(data, CVE5_SCHEMA_PATH)
+        if _result is False:
+            _errors.append(_error)
+
+    return _errors
+
+
+def query_publisher_db(query):
+    '''Function to query the DB created by Content Migration Tool.
+    Args:
+        query (str): Query to send to the DB.
+    Returns:
+        result (list): Query results, empty if no query was executed or no results were returned.
+    '''
+    result = []
+
+    try:
+        cnx = mysql.connector.connect(
+            host="localhost",
+            user="test",
+            password="Test123$",
+            database="test_database"
+            )
+    except mysql.connector.Error as err:
+        cnx = None
+        if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
+            logger.error("Something is wrong with your user name or password")
+        elif err.errno == errorcode.ER_BAD_DB_ERROR:
+            logger.error("Database does not exist")
+        else:
+            logger.error(err)
+
+    if cnx is not None:
+        cursor = cnx.cursor()
+        cursor.execute(query)
+        result = cursor.fetchall()
+        cnx.close()
+
+    return result
+
+
+def clean_migration_tool_output_files():
+    '''Remove all files generated by Content Migration Tool.
+    '''
+
+    def remove_files_in_folder(folders_list):
+        for folder in folders_list:
+            files = glob.glob(os.path.join(folder, '*'))
+            for file in files:
+                delete_file(file)
```

Comment:
We need to sync about it to clarify some doubts.

Reply:
I should move this function to `wazuh_testing.tools.file` because it's not specific to this solution or library.

Reply:
Done in a57d70fff